lib/internal/util.js

```diff
@@ -186,4 +197,27 @@ exports.convertToValidSignal = function convertToValidSignal(signal) {
   }
 
   throw new Error('Unknown signal: ' + signal);
+}
+
+module.exports = {
```

Comment:
Avoiding the need to refer to `exports.{whatever}` is important and is why the various exported functions are all named top level functions (they can refer to each other without the `exports.` bit). That said, it's harmless to add.